services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None,
+
+) -> QuerySet:
+    movie = Movie.objects.all()
```

Comment:
When you operate on multiple objects, it is better to use plural form in variable name
```suggestion
    movies = Movie.objects.all()
```